tests/test_archetypes.py

```
@@ -0,0 +1,548 @@
+            # With modifiers
+            baseline2 = baseline_from_archetype(archetype_name, modifiers={"PLAY": 0.1})
+            assert len(baseline2) == 7
+            for value in baseline2:
+                assert 0.0 <= value <= 1.0
+    
+    def test_affect_map_completeness(self):
+        """AFFECT_MAP should cover all baseline indices."""
+        for archetype in ARCHETYPES.values():
+            baseline = archetype["baseline"]
+            # Should be able to reference all elements via AFFECT_MAP
+            for affect, idx in AFFECT_MAP.items():
```

Ruff (0.15.0)

[warning] Loop control variable `affect` not used within loop body

Rename unused `affect` to `_affect`

(B007)